camelot/io.py

```diff
@@ -79,6 +79,8 @@ def read_pdf(filepath, pages='1', flavor='lattice', suppress_warnings=False,
         PDFMiner char_margin, line_margin and word_margin.
 
         For more information, refer `PDFMiner docs <https://euske.github.io/pdfminer/>`_.
+    password* : str, optional (default: None)
```

Comment:
Can you remove the `*` since this kwarg can be used with both flavors.

Reply:
What is the difference between owner and user pass?

Reply:
Another good catch here. Done.

The difference between Owner and User pass is [described](https://forums.adobe.com/thread/852787) as follows:

> The user password, if set, is what you need to provide in order to open a PDF. Acrobat/Reader will prompt a user to enter the user password. If it's not correct, the document will not open.
>
>The owner password, if set, controls permissions, such as printing, editing, extracting, commenting, etc. Acrobat/Reader will disallow these things based on the permission settings. Acrobat will require this password if you want to set/change permissions.

So user passwords may have fewer permissions, while owner passwords have free-reign.

Reply:
Oh, does PyPDF differentiate between owner and user pass on its own? (given it is just passed as an argument to `decrypt`)

Reply:
PyPDF does handle both cases separately and the type of authentication is returned by the API in the return value (0 = failed, 1 = user, 2 = owner). The concept should be familiar to people generating password protected PDFs, but may not be for general consumers